File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/mr/nu/newbornveinnur/d/NewBornVeinNurRecordDO.java
package iih.ci.mr.nu.newbornveinnur.d;

import xap.sys.appfw.orm.desc.DescManager;
import xap.mw.coreitf.c.GlobalConst;
import xap.mw.coreitf.i.IDODesc;
import xap.mw.coreitf.d.*;
import xap.mw.core.data.*;
import iih.ci.mr.nu.newbornveinnur.d.desc.NewBornVeinNurRecordDODesc;
import java.math.BigDecimal;

/**
 * 新生儿脐静脉护理记录 DO数据 
 * 
 */
public class NewBornVeinNurRecordDO extends BaseDO {
	private static final long serialVersionUID = 1L;

	public static final String ID_NBVNREC= "Id_nbvnrec";
	public static final String ID_NBVN= "Id_nbvn";
	public static final String D_REC= "D_rec";
	public static final String T_REC= "T_rec";
	public static final String REDSWOLLEN= "Redswollen";
	public static final String BLEEDING= "Bleeding";
	public static final String ID_SMOOTH_PIPE= "Id_smooth_pipe";
	public static final String SD_SMOOTH_PIPE= "Sd_smooth_pipe";
	public static final String CATHETER_INSERTION= "Catheter_insertion";
	public static final String CATHETER_EXPOSED= "Catheter_exposed";
	public static final String REPLACE_DRESSING= "Replace_dressing";
	public static final String REPLACE_INFUSION_CONECTOR= "Replace_infusion_conector";
	public static final String OTHER= "Other";
	public static final String ID_SIGN= "Id_sign";
	public static final String CREATEDBY= "Createdby";
	public static final String CREATEDTIME= "Createdtime";
	public static final String MODIFIEDBY= "Modifiedby";
	public static final String MODIFIEDTIME= "Modifiedtime";
	public static final String NAME_SMOOTH_PIPE= "Name_smooth_pipe";
	public static final String NAME_SIGN= "Name_sign";
	public static final String DS = "Ds";
	public static final String SV = "Sv";
	
	public String getId_nbvnrec() {
		return ((String) getAttrVal("Id_nbvnrec"));
	}	
	public void setId_nbvnrec(String Id_nbvnrec) {
		setAttrVal("Id_nbvnrec", Id_nbvnrec);
	}
	public String getId_nbvn() {
		return ((String) getAttrVal("Id_nbvn"));
	}	
	public void setId_nbvn(String Id_nbvn) {
		setAttrVal("Id_nbvn", Id_nbvn);
	}
	public FDate getD_rec() {
		return ((FDate) getAttrVal("D_rec"));
	}	
	public void setD_rec(FDate D_rec) {
		setAttrVal("D_rec", D_rec);
	}
	public FTime getT_rec() {
		return ((FTime) getAttrVal("T_rec"));
	}	
	public void setT_rec(FTime T_rec) {
		setAttrVal("T_rec", T_rec);
	}
	public Integer getRedswollen() {
		return ((Integer) getAttrVal("Redswollen"));
	}	
	public void setRedswollen(Integer Redswollen) {
		setAttrVal("Redswollen", Redswollen);
	}
	public Integer getBleeding() {
		return ((Integer) getAttrVal("Bleeding"));
	}	
	public void setBleeding(Integer Bleeding) {
		setAttrVal("Bleeding", Bleeding);
	}
	public String getId_smooth_pipe() {
		return ((String) getAttrVal("Id_smooth_pipe"));
	}	
	public void setId_smooth_pipe(String Id_smooth_pipe) {
		setAttrVal("Id_smooth_pipe", Id_smooth_pipe);
	}
	public String getSd_smooth_pipe() {
		return ((String) getAttrVal("Sd_smooth_pipe"));
	}	
	public void setSd_smooth_pipe(String Sd_smooth_pipe) {
		setAttrVal("Sd_smooth_pipe", Sd_smooth_pipe);
	}
	public Integer getCatheter_insertion() {
		return ((Integer) getAttrVal("Catheter_insertion"));
	}	
	public void setCatheter_insertion(Integer Catheter_insertion) {
		setAttrVal("Catheter_insertion", Catheter_insertion);
	}
	public Integer getCatheter_exposed() {
		return ((Integer) getAttrVal("Catheter_exposed"));
	}	
	public void setCatheter_exposed(Integer Catheter_exposed) {
		setAttrVal("Catheter_exposed", Catheter_exposed);
	}
	public Integer getReplace_dressing() {
		return ((Integer) getAttrVal("Replace_dressing"));
	}	
	public void setReplace_dressing(Integer Replace_dressing) {
		setAttrVal("Replace_dressing", Replace_dressing);
	}
	public Integer getReplace_infusion_conector() {
		return ((Integer) getAttrVal("Replace_infusion_conector"));
	}	
	public void setReplace_infusion_conector(Integer Replace_infusion_conector) {
		setAttrVal("Replace_infusion_conector", Replace_infusion_conector);
	}
	public String getOther() {
		return ((String) getAttrVal("Other"));
	}	
	public void setOther(String Other) {
		setAttrVal("Other", Other);
	}
	public String getId_sign() {
		return ((String) getAttrVal("Id_sign"));
	}	
	public void setId_sign(String Id_sign) {
		setAttrVal("Id_sign", Id_sign);
	}
	public String getCreatedby() {
		return ((String) getAttrVal("Createdby"));
	}	
	public void setCreatedby(String Createdby) {
		setAttrVal("Createdby", Createdby);
	}
	public FDateTime getCreatedtime() {
		return ((FDateTime) getAttrVal("Createdtime"));
	}	
	public void setCreatedtime(FDateTime Createdtime) {
		setAttrVal("Createdtime", Createdtime);
	}
	public String getModifiedby() {
		return ((String) getAttrVal("Modifiedby"));
	}	
	public void setModifiedby(String Modifiedby) {
		setAttrVal("Modifiedby", Modifiedby);
	}
	public FDateTime getModifiedtime() {
		return ((FDateTime) getAttrVal("Modifiedtime"));
	}	
	public void setModifiedtime(FDateTime Modifiedtime) {
		setAttrVal("Modifiedtime", Modifiedtime);
	}
	public String getName_smooth_pipe() {
		return ((String) getAttrVal("Name_smooth_pipe"));
	}	
	public void setName_smooth_pipe(String Name_smooth_pipe) {
		setAttrVal("Name_smooth_pipe", Name_smooth_pipe);
	}
	public String getName_sign() {
		return ((String) getAttrVal("Name_sign"));
	}	
	public void setName_sign(String Name_sign) {
		setAttrVal("Name_sign", Name_sign);
	}

	public Integer getDs() {
		return ((Integer) getAttrVal(GlobalConst.DELETE_SIGN));
	}
	
	public void setDs(Integer ds) {
		setAttrVal(GlobalConst.DELETE_SIGN, ds);
	}
	
	public FDateTime getSv() {
		return ((FDateTime) getAttrVal(GlobalConst.SYSTEM_VERSION));
	}
	
	public void setSv(FDateTime sv) {
		setAttrVal(GlobalConst.SYSTEM_VERSION, sv);
	}

//	@Override
//	public java.lang.String getParentPKFieldName() {
//		return null;
//	}
  
	@Override
	public String getPKFieldName() {
		return "Id_nbvnrec";
	}
	
	@Override
	public String getTableName() {	  
		return "ci_mr_nu_nbvn_rec";
	}
	
	@Override
	public IDODesc getDODesc() {
		return DescManager.getInstance().getDODesc(NewBornVeinNurRecordDODesc.class);
	}
	
}

File: client/iih_stub/iih.ci_stub/iih.ci.mr/nu/newbornveinnur/d/NewBornVeinNurRecordDO.cs

using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;
using System.Collections.Generic;

namespace iih.ci.mr.nu.newbornveinnur.d
{
    /// <summary>
    /// NewBornVeinNurRecordDO 的摘要说明。
    /// </summary>
    public class NewBornVeinNurRecordDO : BaseDO {

		public const string TABLE_NAME = "ci_mr_nu_nbvn_rec";
		public const string TABLE_ALIAS_NAME = "a1";

        public NewBornVeinNurRecordDO() {
        }
		public string Id_nbvnrec {
            get { return getAttrVal<string>("Id_nbvnrec",null); }
            set { setAttrVal<string>("Id_nbvnrec", value); }
        }
		public string Id_nbvn {
            get { return getAttrVal<string>("Id_nbvn",null); }
            set { setAttrVal<string>("Id_nbvn", value); }
        }
		public DateTime? D_rec {
            get { return getAttrVal<FDate>("D_rec",null); }
            set { setAttrVal<FDate>("D_rec", value); }
        }
		public DateTime? T_rec {
            get { return getAttrVal<FTime>("T_rec",null); }
            set { setAttrVal<FTime>("T_rec", value); }
        }
		public int? Redswollen {
            get { return getAttrVal<int?>("Redswollen",null); }
            set { setAttrVal<int?>("Redswollen", value); }
        }
		public int? Bleeding {
            get { return getAttrVal<int?>("Bleeding",null); }
            set { setAttrVal<int?>("Bleeding", value); }
        }
		public string Id_smooth_pipe {
            get { return getAttrVal<string>("Id_smooth_pipe",null); }
            set { setAttrVal<string>("Id_smooth_pipe", value); }
        }
		public string Sd_smooth_pipe {
            get { return getAttrVal<string>("Sd_smooth_pipe",null); }
            set { setAttrVal<string>("Sd_smooth_pipe", value); }
        }
		public int? Catheter_insertion {
            get { return getAttrVal<int?>("Catheter_insertion",null); }
            set { setAttrVal<int?>("Catheter_insertion", value); }
        }
		public int? Catheter_exposed {
            get { return getAttrVal<int?>("Catheter_exposed",null); }
            set { setAttrVal<int?>("Catheter_exposed", value); }
        }
		public int? Replace_dressing {
            get { return getAttrVal<int?>("Replace_dressing",null); }
            set { setAttrVal<int?>("Replace_dressing", value); }
        }
		public int? Replace_infusion_conector {
            get { return getAttrVal<int?>("Replace_infusion_conector",null); }
            set { setAttrVal<int?>("Replace_infusion_conector", value); }
        }
		public string Other {
            get { return getAttrVal<string>("Other",null); }
            set { setAttrVal<string>("Other", value); }
        }
		public string Id_sign {
            get { return getAttrVal<string>("Id_sign",null); }
            set { setAttrVal<string>("Id_sign", value); }
        }
		public string Createdby {
            get { return getAttrVal<string>("Createdby",null); }
            set { setAttrVal<string>("Createdby", value); }
        }
		public DateTime? Createdtime {
            get { return getAttrVal<FDateTime>("Createdtime",null); }
            set { setAttrVal<FDateTime>("Createdtime", value); }
        }
		public string Modifiedby {
            get { return getAttrVal<string>("Modifiedby",null); }
            set { setAttrVal<string>("Modifiedby", value); }
        }
		public DateTime? Modifiedtime {
            get { return getAttrVal<FDateTime>("Modifiedtime",null); }
            set { setAttrVal<FDateTime>("Modifiedtime", value); }
        }
		public string Name_smooth_pipe {
            get { return getAttrVal<string>("Name_smooth_pipe",null); }
            set { setAttrVal<string>("Name_smooth_pipe", value); }
        }
		public string Name_sign {
            get { return getAttrVal<string>("Name_sign",null); }
            set { setAttrVal<string>("Name_sign", value); }
        }
        public int Ds {
            get { return getAttrVal<int>("Ds",0);}
            set { setAttrVal<int>("Ds", value); }
        }

        public DateTime? Sv        {
            get { return getAttrVal<FDateTime>("Sv",null); }
            set { setAttrVal<FDateTime>("Sv", value); }
        }
        
        /// <summary>
        /// 返回表名
        /// </summary>
        /// <returns></returns>
        public override string getTableName()
        {
            return "ci_mr_nu_nbvn_rec";
        }
        
        /// <summary>
        /// 返回表别名
        /// </summary>
        /// <returns></returns>
        public override string getAliasTableName()
        {
            return "a1";
        }
        
        /// <summary>
        /// 返回主键字段名
        /// </summary>
        /// <returns></returns>
        public override string getPKFieldName()
        {
            return "Id_nbvnrec";
        }

        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.mr.nu.newbornveinnur.d.NewBornVeinNurRecordDO";
        }

        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{"Ds","Sv",  "Id_nbvnrec", "Id_nbvn", "D_rec", "T_rec", "Redswollen", "Bleeding", "Id_smooth_pipe", "Sd_smooth_pipe", "Catheter_insertion", "Catheter_exposed", "Replace_dressing", "Replace_infusion_conector", "Other", "Id_sign", "Createdby", "Createdtime", "Modifiedby", "Modifiedtime", "Name_smooth_pipe", "Name_sign"};
        }
        
    }
}
